### PR TITLE
gitui: update to 0.22.1.

### PR DESCRIPTION
**Pull request by jcgruenhage**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by cinerea0** [comment]
Testing this myself and it works well, it fixed the performance regression in v0.22.0.

**Comment by jcgruenhage** [comment]
@cinerea0 do you use this? I don't really, I just update it because I'm currently listed as the maintainer. If you actually use it, you might be better suited to those maintainer duties though

File: srcpkgs/gitui/template [hunk 1]
```diff
@@ -11,7 +11,7 @@ license="MIT"
 homepage="https://github.com/extrawurst/gitui"
 changelog="https://github.com/extrawurst/gitui/blob/master/CHANGELOG.md"
```
Comment (paper42) [inline srcpkgs/gitui/template:12]:
```suggestion
changelog="https://raw.githubusercontent.com/extrawurst/gitui/master/CHANGELOG.md"
```

**Comment by cinerea0** [comment]
@jcgruenhage Yes, I do use this. If you don't want to be the maintainer anymore, I'd be happy to take ownership.

**Comment by jcgruenhage** [comment]
Pushed a new commit setting you as the maintainer ^^

**Pull request merged**